libobjrenderer/src/objectsscene.cpp

```diff
@@ -371,64 +371,37 @@ void ObjectsScene::setSceneRect(const QRectF &rect)
 	QGraphicsScene::setSceneRect(0, 0, rect.width(), rect.height());
 }
 
-QRectF ObjectsScene::itemsBoundingRect(bool seek_only_db_objs, bool selected_only)
+QRectF ObjectsScene::itemsBoundingRect(bool override, bool selected_only, QList<QGraphicsItem *> objs)
```

Comment:
Changing the itemsBoundingRect is really critical... There are some important portions that refers to it so I'll test carefully. I did some quick tests with your changes and got some problems, for example, the bounding rect used by the exporter was incorrect and the resulting png file was bigger than expected. So, this kind of change demands a lot of tests.